### PR TITLE
feat: improve wallets loading

### PR DESCRIPTION
**Pull request by chybisov**
Development env

image
image

Production env

image
image

Also, fixes [LF-9498](https://lifi.atlassian.net/browse/LF-9498)

### CODE REVIEW

File: src/components/Menus/WalletSelectMenu/WalletSelectMenu.tsx [hunk 1]
```diff
@@ -48,49 +47,56 @@ export const WalletSelectMenu = ({ anchorEl }: MenuProps) => {
       height: 72,
     },
   };
-
+  const anyWidnow = typeof window !== 'undefined' ? (window as any) : undefined;
```
Comment (oktapodia) [inline src/components/Menus/WalletSelectMenu/WalletSelectMenu.tsx:50]: Typo

File: src/config/metaMask.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { siteName } from '@/app/lib/metadata';
+import type { MetaMaskParameters } from 'wagmi/connectors';
+
+export const defaultMetaMaskConfig: MetaMaskParameters = {
+  dappMetadata: {
+    name: siteName,
+    url:
+      typeof window !== 'undefined'
```
Comment (oktapodia) [inline src/config/metaMask.ts:8]: Can you use process.env.NEXT_SITE_URL instead?

**Review by oktapodia — CHANGES_REQUESTED** [review]
Few comments

File: src/components/Menus/WalletSelectMenu/WalletSelectMenu.tsx (hunk 1 above)
Comment (chybisov) [reply src/components/Menus/WalletSelectMenu/WalletSelectMenu.tsx:50]: it'll be removed, just for test :)

**Pull request merged**